code/modules/antagonists/changeling/powers/armblade_stinger.dm

```diff
@@ -0,0 +1,108 @@
+	AddComponent(/datum/component/butchering, 60, 80)
+
+/obj/item/melee/stingblade/afterattack(atom/target, mob/user, proximity)
+	. = ..()
+	if(!proximity || !iscarbon(M))
+		return
+		
+	//inject carbon entities by default
+	imp_in.reagents.add_reagent(/datum/reagent/medical/tirizene, 2)
+	log_combat(user, M, "injected", src, "5u tirizene")
+	
+	if(!reagents.total_volume) //it is empty
+		return
+
+	//Always log attemped injects for admins
+	var/list/injected = list()
+	for(var/datum/reagent/R in reagents.reagent_list)
+		injected += R.name
+	var/contained = english_list(injected)
+	log_combat(user, M, "attempted to inject", src, "([contained])")
+
+	if(reagents.total_volume && M.can_inject(user, 1)) 
+		
+		var/fraction = min(amount_per_transfer_from_this/reagents.total_volume, 1)
+		reagents.reaction(M, INJECT, fraction)
+		if(M.reagents)
+			var/trans = reagents.copy_to(M, amount_per_transfer_from_this)
+			to_chat(user, "<span class='notice'>[trans] unit\s injected.  [reagents.total_volume] unit\s remaining in [src].</span>")
+			log_combat(user, M, "stingblade injected", src, "([contained])")
+	
+	//default armblade functionality
+	if(istype(target, /obj/structure/table))
+		var/obj/structure/table/T = target
+		T.deconstruct(FALSE)
+
+	else if(istype(target, /obj/machinery/computer))
+		var/obj/machinery/computer/C = target
+		C.attack_alien(user) //muh copypasta
+		
+	//Maybe remove the functionality of the armblade, given this is not actually an armblade
+	/*else if(istype(target, /obj/machinery/door/airlock))
+		var/obj/machinery/door/airlock/A = target
+
+		if((!A.requiresID() || A.allowed(user)) && A.hasPower()) //This is to prevent stupid shit like hitting a door with an arm blade, the door opening because you have acces and still getting a "the airlocks motors resist our efforts to force it" message, power requirement is so this doesn't stop unpowered doors from being pried open if you have access
+			return
+		if(A.locked)
+			to_chat(user, "<span class='warning'>The airlock's bolts prevent it from being forced!</span>")
+			return
+
+		if(A.hasPower())
+			user.visible_message("<span class='warning'>[user] jams [src] into the airlock and starts prying it open!</span>", "<span class='warning'>We start forcing the [A] open.</span>", \
+			"<span class='italics'>You hear a metal screeching sound.</span>")
+			playsound(A, 'sound/machines/airlock_alien_prying.ogg', 100, 1)
+			if(!do_after(user, 100, target = A))
+				return
+		//user.say("Heeeeeeeeeerrre's Johnny!")
+		user.visible_message("<span class='warning'>[user] forces the airlock to open with [user.p_their()] [src]!</span>", "<span class='warning'>We force the [A] to open.</span>", \
+		"<span class='italics'>You hear a metal screeching sound.</span>")
+		A.open(2)*/
+	
+/obj/item/melee/stingblade/is_refillable()	
+	. = ..()
```

Comment:
This is pointless

Reply:
> This is pointless

How come? It should say true, so that it can be filled with chemicals, no?

Reply:
Because you are overriding the base proc to call the base proc, which would be the same as if that whole proc wasn't there at all.

Reply:
Should I maybe add a return TRUE? Or is that by default?